keras_cv/bounding_box/utils.py

```diff
@@ -87,3 +87,104 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def pad_with_sentinels(bounding_boxes, default_value=-1):
+    """Pads the given bounding box tensor with -1s.
+
+    This is done to convert RaggedTensors to be converted into
+    standard Dense tensors, which have better performance and
+    compatibility within the TensorFlow ecosystem.
+
+    Args:
+        bounding_boxes: a ragged tensor of bounding boxes.
+        Can be batched or unbatched.
+
+    Returns:
+        a Tensor containing the -1 padded bounding boxes.
+    """
+    return bounding_boxes.to_tensor(default_value)
+
+
+def filter_sentinels(bounding_boxes, default_value=-1):
+    """converts a Dense padded bounding box `Tensor` to a `tf.RaggedTensor`.
```

Comment:
"""converts a Dense padded bounding box `tf.Tensor` to a `tf.RaggedTensor`.

Reply:
@divyashreepathihalli I see the review request is back but it looks like these 3 issues are still unresolved - GitHub can sometimes hide issues in big reviews even if un-resolved, so make sure to click "load hidden comments" before re-requesting!

Reply:
### sorry about that